_data/simple-icons.json

```diff
@@ -7370,6 +7370,11 @@
             "hex": "8C5AFF",
             "source": "https://hyperskill.org"
         },
+        {
+            "title": "HyperX",
+            "hex": "E21836",
+            "source": "https://ca.hyperx.com/cdn/shop/files/image_placeholder.svg?v=15046939785134493775"
```

Comment:
```suggestion
            "source": "https://ca.hyperx.com/cdn/shop/files/image_placeholder.svg"
```

Reply:
You icon is still in a wrong size, this needs to be fixed.

Reply:
Fixed it.